lib/hdclient.js

```diff
@@ -105,6 +107,20 @@ class HyperdriveClient {
         this.errorAgent = new kafka.Producer(kafkaOptions);
     }
 
+    /**
+     * Load UUID -> host:port mapping
+     * Use passed object or load JSON file
+     *
+     * @param {String|Object} uuidmapping - Mapping UUID on host:port
+     * @return {undefined}
+     */
+    setupUuidMapping(uuidmapping) {
+        if (typeof(uuidmapping) === 'string') {
+            const loaded = fs.readFileSync(uuidmapping);
+            this.options.uuidmapping = JSON.parse(loaded);
+        }
```

Comment:
no error management if uuidmapping is not of type string ?
assert ?

Reply:
If it is not a stirng, I assume this.options.uuidmapping already contains a valid JS object doing the ampping. Not super proud of this code though

Reply:
Okay :+1: 
So a comment can be good :)